subtrack.Tests/SubscriptionCalculatorTests/IntegrationTestBase.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Data.Common;
+using System.Linq;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using subtrack.DAL;
+using subtrack.DAL.Entities;
+using Xunit;
+using subtrack.MAUI.Services;
+
+namespace EF.Testing.UnitTests;
+
+public class SqliteInMemoryBloggingControllerTest : IDisposable
+{
+    private readonly DbConnection _connection;
```

Comment:
It'd be nice if the database was simply created in the ctor and assigned to a `protected` field. Then inside the dispose method you can call the `_dbContext.Dispose()` instead of the connection and whoever inherits this class won't need to actually implement the dispose method or make use of a using method and the "CreateContext()" method can be removed.